doc/usage.rst

```diff
@@ -358,6 +358,20 @@ LG_PROXY
 Specifies a SSH proxy host to be used for port forwards to access the
 coordinator and network resources.
 
+It is worth noting that each NetworkService will define its own proxy host in
+the `params.extra.proxy` variable, which will take precedence over `LG_PROXY`.
+The NetworkService `params.extra.proxy` will be used when `LG_PROXY` is set or
```

Comment:
This is not only true for the NetworkService, but for every resource which uses `get_host_and_port` from the `labgrid.util.proxy`. Labgrid always assumes that exporters are accessible via a configured SSH config. Indeed the isolated flag enforces that all connections to resources from the isolated exporter are tunneled through the exporter instead of the `LG_PROXY` host. I think the information is currently available in the docs, but not consolidated enough to understand the full picture easily.

Reply:
You're right, the `--isolator` mechanism and SSH config is explained in the [Proxy Mechanism section](https://github.com/labgrid-project/labgrid/blob/master/doc/overview.rst#proxy-mechanism).

I was expecting labgrid to use LG_PROXY for client-to-coordinator and client-to-exporter proxies. The current documentation suggests that LG_PROXY configures both, and that labgrid will only prefer the client-to-exporter proxy if the exporter is isolated.

In my case, an "isolated" exporter was not configured with `--isolated` and setting LG_PROXY selects the client-to-coordinator proxy (a nickname I have in .ssh/config), and activated the client-to-exporter proxy - a configuration I hadn't set up. I think the right configuration for my arrangement is:
1. `labgrid-exporter --isolated', with the exporter host in SSH config, as labgrid expects.
2. Configure client-to-coordinator with either LG_PROXY or LG_CROSSBAR, with coordinator host also in SSH config.

I've made another set of doc changes to make the current behaviour more clear.